product-service/handlers/get-products/index.ts

```diff
@@ -0,0 +1,19 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from '../../mocks/products';
+
+export const handler: APIGatewayProxyHandler = async () => {
```

Comment:
Could you please provide an URL to your CloudFront-deployed Frontend application, so I can make sure this route is integrated into your Frontend app?